src/App.jsx

```diff
@@ -37,21 +41,21 @@ export function App() {
   var instructions = appInfo.instructions.start
   
   var currentMenuItems = [];
-  if (categ !== '' && res === ''){
+  if (isStringEmpty(categ) && isStringEmtpy(res)) {
     instructions = appInfo.instructions.onlyCategory
   }
-  if (res !== '' && categ === ''){
+  if (isStringEmtpy(res) && isStringEmtpy(categ)){
```

Comment:
Add a space after curly braces
```suggestion
  if (isStringEmtpy(res) && isStringEmtpy(categ)) {
```